apps/voting/app/src/utils.js

```diff
@@ -6,3 +6,15 @@ export function pluralize(count, singular, plural, re = PLURALIZE_RE) {
   }
   return plural.replace(re, count)
 }
+
+export function makeEtherscanBaseUrl(network) {
+  // Don't make etherscan urls if the network isn't one that etherscan supports
+  if (
+    network === 'mainnet' ||
+    network === 'kovan' ||
+    network === 'rinkeby' ||
+    network === 'ropsten'
+  ) {
+    return `https://${network === 'mainnet' ? '' : `${network}.`}etherscan.io`
+  }
+}
```

Comment:
@bpierre was thinking the same, although I haven't quite been convinced what type of API it would have. Quite hard to make it generic across all the different url types (addresses, blocks, etc), but we could probably have a "custom" path prop that just used whatever you gave it.